app/routes/events.tsx

```diff
@@ -19,32 +21,47 @@ export const meta: V2_MetaFunction = () => {
 };
 
 export default function Events() {
-	const data = useLoaderData<typeof loader>();
+	const data = useLoaderData<
+		Record<
+			number,
+			{
+				date: string;
+				link: string;
+				location: string;
+				topics: string[];
+			}[]
+		>
+	>();
 
 	return (
 		<PageGrid
 			left={
-				<ol className="events-year">
-					{Object.entries(data)
-						.sort(([a], [b]) => +b - +a)
-						.map(([year, events]) => (
-							<li className="events-year-list" key={year}>
-								<h2 className="events-year-heading">{year}</h2>
-								<ol className="events-year-events-list">
-									{events.map((event) => (
-										<EventDetails
-											date={new Date(event.date)}
-											key={event.date}
-											link={event.link}
-											linkText="Details"
-											location={event.location}
-											topics={event.topics}
-										/>
-									))}
-								</ol>
-							</li>
-						))}
-				</ol>
+				<>
+					<a href={`webcal://${site.domain}/ics-feed.ics`} rel="noreferrer">
```

Comment:
I think we'll need to add signposting here to make sure folks don't just download this file and expect it to be in sync; it would also be nice if we could add a "subscribe with your Google calendar" link here, too